WebRoot/js/zimbraMail/briefcase/controller/ZmBriefcaseController.js

```diff
@@ -485,6 +489,21 @@ ZmBriefcaseController.prototype._doDelete = function(items, hardDelete) {
 	dialog.popup(message, this._doDelete2.bind(this, items, hardDelete));
 };
 
+/**
+ * Check if deleting item(s) is opened
+ *
+ * @param {Array}   toDelete   list of items to delete
+ */
+ZmBriefcaseController.prototype.isDeletingItemOpen =
+function(toDelete) {
+	var deletingItemIds = [];
+	for (var i = 0; i < toDelete.length; i++) {
+		deletingItemIds.push(toDelete[i].id);
+	}
+	var appViewMgr = appCtxt.getAppViewMgr();
+	return appViewMgr && appViewMgr.isDeletingItemOpen(deletingItemIds);
```

Comment:
I think we don't need to check `appViewMgr`. Can we directly call `isDeletingItemOpen`?.  Same for other places as well where we are checking the `appViewMgr`.

```suggestion
	return appCtxt.getAppViewMgr().isDeletingItemOpen(deletingItemIds);
```

Reply:
fixed. Thank you for pointing it out.